frontend/src/app/error.tsx

```diff
@@ -7,22 +7,36 @@ import { SyncIcon } from "@primer/octicons-react"
 import Button from "@/components/Button"
 import ErrorBoundary from "@/components/ErrorBoundary"
 import SetPageTitle from "@/components/SetPageTitle"
+import { RequestFailedError } from "@/lib/api"
 
-export const metadata = {
-    title: "Error",
-}
+type ErrorPageProps = {error: Error, reset: () => void };
 
-export default function Error({
-    error,
-    reset,
-}: {
-    error: Error
-    reset: () => void
-}) {
-    useEffect(() => {
-        console.error(error)
-    }, [error])
+function NetworkErrorPage({ error, reset }: ErrorPageProps) {
+    return <>
+        <SetPageTitle title="Error" />
+        <div className="grow" />
+        <main className="max-w-prose p-4 md:mx-auto">
+            <h1 className="py-4 text-3xl font-semibold">We're having some trouble reaching the backend</h1>
 
+            <div className="rounded bg-gray-2 p-4 text-gray-11">
+                <code className="font-mono text-sm">{error.toString()}</code>
+            </div>
+
+            <p className="py-4">
+                If your internet connection is okay, we're probably down for maintenence, and will be back shortly. If this issue persists - <a href="https://discord.gg/sutqNShRRs" className="text-blue-11 hover:underline active:translate-y-px">let us know</a>.
```

Comment:
```suggestion
                If your internet connection is okay, we're probably down for maintenance, and will be back shortly. If this issue persists - <a href="https://discord.gg/sutqNShRRs" className="text-blue-11 hover:underline active:translate-y-px">let us know</a>.
```

Reply:
that word forever feels incorrectly spelled

Reply:
🤦‍♂️